modules/base/manifests/ntp.pp

```diff
@@ -0,0 +1,21 @@
+class base::ntp (
+  $ntp_service = 'ntpd',
```

Comment:
In class [base] you are using facts $::osfamily for getting $ntp_service vars. But in this class you just hardcode this var.